test/functional/plugin/lsp/utils_spec.lua

```diff
@@ -0,0 +1,74 @@
+local helpers = require('test.functional.helpers')(after_each)
+
+local eq = helpers.eq
+local exec_lua = helpers.exec_lua
+
+describe('vim.lsp.util', function()
+  before_each(helpers.clear)
+  after_each(helpers.clear)
```

Comment:
This is not needed.
```suggestion
```

Reply:
I've removed it, but would you mind clarifying why this is not needed? Is is because other tests already run `helpers.clear` `before_each` test?

Reply:
Yes

Reply:
Thanks